logstash-core/lib/logstash/config/mixin.rb

```diff
@@ -147,7 +147,7 @@ def plugin_status(status = nil)
     # inside the gemspec.
     def milestone(m = nil)
       @logger = Cabin::Channel.get(LogStash)
-      @logger.warn(I18n.t('logstash.plugin.deprecated_milestone', :plugin => config_name))
+      @logger.debug(I18n.t('logstash.plugin.deprecated_milestone', :plugin => config_name))
```

Comment:
Well, From the issue I understood that this was confusing people, since it doesn't really interfere with the plugins. I though it was more of a developer focus message than a user focus message this is why I went ahead with debug?

This was my reasoning behind it.